Sources/Brave/Frontend/Browser/Search/Recent Search/RecentSearchHeaderView.swift

```diff
@@ -209,4 +209,17 @@ class RecentSearchHeaderView: UICollectionReusableView {
 
     themeViews()
   }
+  
+  public override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+    let adjustedBounds = CGRect(x: hideClearButton.center.x,
+                                y: hideClearButton.center.y,
+                                width: 0.0,
+                                height: 0.0).inset(by: UIEdgeInsets(top: -25.0, left: -25.0, bottom: -5.0, right: -25.0))
```

Comment:
Why not use the button bounds instead of center + 0 width/height?

Reply:
Fixed. I used frame instead because bounds won't return the correct location within the parent.